rosetta-source/src/main/rosetta/mapping-fpml-confirmation-tradestate-synonym.rosetta

```diff
@@ -1239,9 +1243,9 @@ synonym source FpML_5_Confirmation_To_TradeState extends FpML
 
     Trigger:
         + level
-            [value "level"]
-        + levelPercentage
-            [value "levelPercentage"]
+            [hint "level"]
+            [hint "levelPercentage"]
+            [value "levelPrice"]
```

Comment:
Is levelPrice actually part of the FpML schema?

Reply:
yes this is part of the FpML schema, I am just covering it to be safe in the future
![image](https://github.com/finos/common-domain-model/assets/109520965/67ce43a9-d503-43e5-aaaf-17b98bbfa115)

Reply:
OK great, thanks for checking